build.fsx

```diff
@@ -178,55 +178,67 @@ Target.create "ReleaseGitHub" (fun _ ->
 
     let client =
         let token =
-           match getBuildParam "GITHUB_TOKEN" with
-           | s when not (isNullOrWhiteSpace s) -> s
-           | _ -> failwith "please set the GITHUB_TOKEN environment variable to a github personal access token with repo access."
+            match getBuildParam "GITHUB_TOKEN" with
+            | s when not (isNullOrWhiteSpace s) -> s
+            | _ ->
+                failwith
+                    "please set the GITHUB_TOKEN environment variable to a github personal access token with repo access."
 
         GitHub.createClientWithToken token
-    let files = !! (nugetDir </> "*.nupkg")
+
+    let files = !!(nugetDir </> "*.nupkg")
 
     // release on github
     let cl =
         client
-        |> GitHub.draftNewRelease gitOwner gitName release.NugetVersion (release.SemVer.PreRelease <> None) release.Notes
-    (cl,files)
+        |> GitHub.draftNewRelease
+            gitOwner
+            gitName
+            release.NugetVersion
+            (release.SemVer.PreRelease <> None)
```

Comment:
again, not better - the cited config has a longer line len so it might leave it unchanged?

Reply:
Yes, I prefer longer lines also so I’ll use the TaskSeq as a base